virtuoso.sh

```diff
@@ -54,4 +58,7 @@ then
     echo `date +%Y-%m-%dT%H:%M:%S%:z` > .data_loaded
 fi
 
-exec virtuoso-t +wait +foreground
+# NOTE: put back Virtuoso's server port to 8890 before starting up
+crudini --set virtuoso.ini HTTPServer ServerPort 8890
```

Comment:
Only the default port is exposed on the container, so if you change it, I don't think you'd be able to actually connect.

I think the general docker way is that inside the container you use the default port, and change this mapping using docker when linking to other containers or binding to the host.